index.html

```diff
@@ -22,5 +22,16 @@
             <div class="line"></div>
         </div>
     </nav>
```

Comment:
- [ ] I think it was a good idea to create a nav link here to contain the list but it would be better to get the `h1` tag outside the nav and create a header element to contain them since it would be more semantic